superset/assets/src/explore/controls.jsx

```diff
@@ -846,6 +841,12 @@ export const controls = {
       const newState = {};
       if (state.datasource) {
         newState.options = state.datasource.columns.filter(c => c.is_dttm);
+        newState.default = null;
```

Comment:
Not related to your PR: `props` would be a better variable name than `newState` in this method.

Reply:
@mistercrunch I've renamed `newState` to `props`.